main-actions/src/main/scala/sbt/ForkTests.scala

```diff
@@ -93,12 +112,20 @@ private[sbt] object ForkTests {
         val acceptorThread = new Thread(Acceptor)
         acceptorThread.start()
 
-        val fullCp = classpath ++: Seq(IO.classLocationFile[ForkMain], IO.classLocationFile[Framework])
-        val options = Seq("-classpath", fullCp mkString File.pathSeparator, classOf[ForkMain].getCanonicalName, server.getLocalPort.toString)
+        val fullCp = classpath ++: Seq(IO.classLocationFile[ForkMain],
+                                       IO.classLocationFile[Framework])
+        val options = Seq("-classpath",
+                          fullCp mkString File.pathSeparator,
+                          classOf[ForkMain].getCanonicalName,
+                          server.getLocalPort.toString)
         val ec = Fork.java(fork, options)
         val result =
           if (ec != 0)
-            TestOutput(TestResult.Error, Map("Running java with options " + options.mkString(" ") + " failed with exit code " + ec -> SuiteResult.Error), Iterable.empty)
+            TestOutput(TestResult.Error,
+                       Map(
+                         "Running java with options " + options
+                           .mkString(" ") + " failed with exit code " + ec -> SuiteResult.Error),
```

Comment:
Yes, I agree. Scalafmt leaves respects line breaks around infix operators. If you put a newline between `+` and `options` the formatting would look nicer.